Mathlib/Order/Booleanisation.lean

```diff
@@ -3,7 +3,8 @@ Copyright (c) 2023 Yaël Dillies. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Yaël Dillies
 -/
-import Mathlib.Order.BooleanAlgebra
+import Mathlib.Order.BooleanAlgebra.Defs
```

Comment:
```suggestion
```
etc etc (probably CI will fail if you don't remove these, `lake exe shake` will surely catch them)